cryptol-remote-api/run_rpc_tests.sh

```diff
@@ -11,6 +11,12 @@ python3 -m venv virtenv
 . virtenv/bin/activate
 pip install -r requirements.txt
 
+mypy cryptol/ tests/
+if [ $? -ne 0 ]; then
+  echo "Code failed to typecheck with mypy"
+  exit 1
+fi
```

Comment:
I would prefer we did not abort the tests entirely if typechecking fails, but rather increment the `NUM_FAILS` variable declared in that file. It is nice to get as much feedback as possible from a test run rather than abort early IMO

Reply:
Good idea. I've done that in my latest push.